main.c

```diff
@@ -71,10 +74,60 @@ static struct kanshi_profile *match(struct kanshi_state *state,
 }
 
 
+static void exec_command(char *cmd) {
+	pid_t pid, child;
+	// Fork process
+	if ((pid = fork()) == 0) {
+		setsid();
+		sigset_t set;
+		sigemptyset(&set);
+		sigprocmask(SIG_SETMASK, &set, NULL);
```

Comment:
Why is this necessary? It makes it impossible to kill the process with e.g. SIGINT, right?

Reply:
Well, this is a quick copy paste from sway's `exec` implementation, so you probably know better than me what this does. I will ave to look into that code again.

Reply:
What these three lines do is clear (reset) the signal mask, so that no signal is blocked anymore, if any was blocked before. See https://man.openbsd.org/sigsetmask.3#EXAMPLES. This clears any special mask that could have been used by kanshi, or that could be used later, for example SIGUSR1, as is the case in sway. (See https://github.com/swaywm/sway/commit/7d8413d9628ed493790454410a28b9f0c41444e6)

It think it belongs here.